SCClassLibrary/JITLib/ProxySpace/BusPlug.sc

```diff
@@ -256,6 +256,7 @@ BusPlug : AbstractFunction {
 			^this
 		};
 		if(bus.rate == \control) { "Can't monitor a control rate bus.".warn; monitor.stop; ^this };
+		group = group ?? {this.homeServer.defaultGroup};
```

Comment:
`?` always evaluates the second operand, whether it will be used or not.

If I recall correctly, `??` inlines the following function if possible, so the cost should be less than pushing a function and calling `value`. In fact, I just checked: in `something ?? { blah }`, `??` compiles to a ControlOpcode that skips over the function's byte codes if the receiver is not nil. The cost of the jump should be a lot less.